plugins/hls-alternate-number-format-plugin/src/Ide/Plugin/Conversion.hs

```diff
@@ -168,10 +167,10 @@ toDecimal :: Integral a => a -> String
 toDecimal = toBase showInt ""
 
 toBinary :: (Integral a, Show a) => a -> String
-toBinary = toBase showBin "0b"
+toBinary = toBase showBin_ "0b"
   where
     -- this is not defined in versions of Base < 4.16-ish
-    showBin = showIntAtBase 2 intToDigit
+    showBin_ = showIntAtBase 2 intToDigit
```

Comment:
Avoiding name shadoing warning.
Technically we could raise the lower bound of base (>= 4.16) and just use showBin from base now, that we only officially support ghc 9.2+, but I went with this less invasive "fix".

Reply:
I think fixing it immediately is fine, as it is still trivial to review.